pkgs/top-level/all-packages.nix

```diff
@@ -5042,6 +5042,8 @@ with pkgs;
 
   sks = callPackage ../servers/sks { inherit (ocamlPackages_4_02) ocaml camlp4; };
 
+  sky = callPackage ../applications/networking/instant-messengers/sky { };
```

Comment:
This needs to be further down in the `### APPLICATIONS` section.
I'd say between `shogun` and `smplayer`